doc/cookbook/source/examples/converter/multidimensional_scaling.rst

```diff
@@ -0,0 +1,54 @@
+========================
+Multidimensional Scaling
+========================
+
+Multidimensional Scaling (MDS) is a set of data analysis methods, which allow one to infer the dimensions of the perceptual space of subjects. The entering data are typically a measure of the global similarity or dissimilarity of objects, called proximities. The outcome is a spatial configuration in which the objects are represented as points arranged in such a way, that their distances correspond to their proximities.
+
+This example features classical MDS, a subset of metric MDS, where the distances in the transformed space preserve the intervals and ratios between the proximities as good as possible. In contrast, nonmetric MDS only reflects the order of the proximities.
+
+See :cite:`Wickelmaier03anintroduction` for a detailed introduction.
+
+In order to find the coordinates :math:`(x_n, y_n)` of :math:`N` items given the proximity matrix :math:`D` with entries the distances :math:`d_{ij}=\sqrt{(x_i -x_j)^2 + (y_i - y_j)^2}` between pairs of items, classical MDS minimizes the following loss function:
+
+.. math::
+    Strain_D(x_1, x_2, \cdots, x_N ) = \frac{\sum_{ij} (b_{ij} -d_{ij})^2}{\sum_{ij} b_{ij}^2}
```

Comment:
"Strain_D" is strange. If you want to use text in math mode, use \text. Also, please dont define objects if you dont use them later